os/kernel/log_dump/log_dump.c

```diff
@@ -125,8 +130,28 @@ static void log_dump_mem_check(size_t max_size)
 	}
 }
 
+size_t log_dump_set(FAR const char *buffer, size_t buflen)
+{
+	(void)buflen;
+
+	if (strncmp(buffer, LOGDUMP_SAVE_START, strlen(LOGDUMP_SAVE_START) + 1) == 0) {
+		is_started_to_save = true;
+	} else if (strncmp(buffer, LOGDUMP_SAVE_STOP, strlen(LOGDUMP_SAVE_STOP) + 1) == 0) {
+		is_started_to_save = false;
+	} else {
+		return LOG_DUMP_OPT_FAIL;
+	}
+	return LOG_DUMP_OK;
+}
+
+
```

Comment:
Could we avoid these multiple extra empty lines please?

Reply:
Removed. Thanks.